java/api/src/main/java/org/ray/api/RayPyActor.java

```diff
@@ -3,7 +3,7 @@
 /**
  * Handle of a Python actor.
  */
-public interface RayPyActor extends RayActor {
+public interface RayPyActor<A> extends RayActor, PyActorCall<A> {
```

Comment:
The generic type `A` shouldn't be needed?

Reply:
I have removed it.